contracts/CharityChallenge.sol

```diff
@@ -73,10 +77,12 @@ contract CharityChallenge {
         address _arbitrator,
         uint256 _timeout,
         uint256 _challengeEndTime,
+        uint8 _makerFee,
         bool _unlockOnNo
     ) public
     {
         require(_npoAddresses.length == _ratios.length);
+        require(_makerFee < 100);
```

Comment:
I think we should use a finer grain to support 2 decimal points like the standard credit card fee 2.99%
So the value of 1% is 100
_makerFee < 10000

https://www.reddit.com/r/ethereum/comments/3s6rfh/best_practices_for_handling_percentages_in/